lib/zipkin-tracer/trace.rb

```diff
@@ -0,0 +1,67 @@
+require 'finagle-thrift'
+require 'finagle-thrift/trace'
+
+module Trace
+  class Span
+    def to_h
+      {
+        name: @name,
+        traceId: @span_id.trace_id.to_s,
+        id: @span_id.span_id.to_s,
+        parentId: @span_id.parent_id.nil? ? nil : @span_id.parent_id.to_s,
```

Comment:
The server is only doing this for portability with legacy instrumentation.
It will eventually stop. In general instrumentation should be sending
complete spans, which can trivially perform this step, and must when they
get to the point of local spans.